crates/transport_tx5/src/lib.rs

```diff
@@ -320,6 +320,17 @@ impl TxImp for Tx5Transport {
         })
     }
 
+    fn get_connected_peers(&self) -> BoxFut<'_, K2Result<Vec<Url>>> {
+        Box::pin(async {
+            Ok(self
+                .ep
+                .get_connected_peer_addresses()
+                .into_iter()
+                .filter_map(|u| Url::from_str(u.as_ref()).ok())
```

Comment:
We don't need ownership here, so I think the move done by `into_iter` is unnecessary. This should be able to become:

```suggestion
                .iter()
                .filter_map(|u| Url::from_str(u).ok())
```

Reply:
Good spot, yes